helix-term/src/commands.rs

```diff
@@ -6015,6 +6006,39 @@ fn jump_to_label(cx: &mut Context, labels: Vec<Range>, behaviour: Movement) {
     });
 }
 
+fn alphabet_char(alphabet: &[char], index: usize) -> Tendril {
+    let mut res = Tendril::new();
+    res.push(alphabet[index]);
+    res
+}
+
+fn labels_to_overlays<'a>(
```

Comment:
This can move within the `jump_to_label` function. See https://github.com/helix-editor/helix/blob/545ff1aac6dfa15bc0c28109a8af1dd6a8d277e0/helix-term/src/commands/lsp.rs#L317-L339 which is similar

Then `alphabet_char` can live within this function as it was prior to this change:

```rust
let alphabet_char = |i| {
    let mut res = Tendril::new();
    res.push(alphabet[i]);
    res
};
```